core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/SbomUtils.java

```diff
@@ -726,10 +826,37 @@ public static void addPedigreeCommit(Component c, String url, String uid) {
 
             c.setPedigree(pedigree);
         }
+
+        Matcher matcher = gitProtocolPattern.matcher(url);
+
+        if (matcher.find()) {
+            log.debug(
+                    "Found URL to be added as pedigree commit with the 'git@' protocol: '{}', trying to convert it into 'https://'",
+                    url);
+
+            url = "https://" + matcher.group(1) + "/" + matcher.group(2);
+
+            log.debug("Converted into: '{}'", url);
+
+        }
+
+        Pedigree pedigree = c.getPedigree() == null ? new Pedigree() : c.getPedigree();
+        List<Commit> commits = new ArrayList<>();
+        if (pedigree.getCommits() != null) {
+            commits.addAll(pedigree.getCommits());
+        }
+
+        Commit newCommit = new Commit();
+        newCommit.setUid(uid);
+        newCommit.setUrl(url);
+        commits.add(newCommit);
+        pedigree.setCommits(commits);
+
+        c.setPedigree(pedigree);
     }
```

Comment:
@vibe13 I tried to rebase to fix merge conflicts. Why is this new code in here? What am I missing?

Reply:
@dwalluck  ? I have no idea, I pushed that code 2 years ago?

Reply:
@vibe13 OK, it should be ready to merge after approval, but please see the note I just added about the URLs.